rm-config/defaults/config.toml

```diff
@@ -17,11 +17,17 @@ headers_hide = false
 [connection]
 url = "http://CHANGE_ME:9091/transmission/rpc" # REQUIRED!
 
+# Refresh timings (in seconds)
+torrents_refresh = 5
+stats_refresh = 5
+free_space_refresh = 10
+
 # If you need username and password to authenticate:
 # username = "CHANGE_ME"
 # password = "CHANGE_ME"
 
-# Refresh timings (in seconds)
-torrents_refresh = 5
-stats_refresh = 10
-free_space_refresh = 10
+
+[torrents_tab]
+# Available fields:
+# Name, SizeWhenDone, Progress, DownloadRate, UploadRate, DownloadDir
+headers = ["Name", "SizeWhenDone", "Progress", "DownloadRate", "UploadRate"]
```

Comment:
maybe rename `SizeWhenDone` to `Size` ?

Reply:
I would love to do that but I'm not sure if it won't become ambiguous in the future, since there's already another [type of size](https://docs.rs/transmission-rpc/latest/transmission_rpc/types/enum.TorrentGetField.html#variant.TotalSize) in the spec